tools/openapi/src/index.js

```
@@ -0,0 +1,148 @@
+const DEFAULT_MARKDOWN = path.join(REPO_ROOT, 'docs', 'REST API', 'rest-api.md');
+const DEFAULT_OUT = path.join(REPO_ROOT, 'build', 'openapi.yml');
+
+function parseArgs(argv) {
+  const args = { out: DEFAULT_OUT, stdout: false, markdown: DEFAULT_MARKDOWN };
+  for (let i = 0; i < argv.length; i++) {
+    const a = argv[i];
+    switch (a) {
+      case '--stdout':
+        args.stdout = true;
+        break;
+      case '--out':
```

ESLint

[error] Replace `'--out'` with `"--out"`

(prettier/prettier)